ui/app/config.js

```diff
@@ -99,6 +110,19 @@ ConfigScreen.prototype.render = function () {
   )
 }
 
+function rpcValidation (newRpc, state) {
+  if (validUrl.isWebUri(newRpc)) {
+    state.dispatch(actions.setRpcTarget(newRpc))
+  } else {
+    var appendedRpc = `http://${newRpc}`
+    if (validUrl.isWebUri(appendedRpc)) {
+      state.dispatch(actions.displayWarning('URIs require the appropriate HTTP/HTTPS prefix.'))
```

Comment:
Oh I see, you aren't changing what they typed, you're just telling them the type of error. Yeah, that's ok with me.  Thanks for clarifying.